8.0/config/conf.d/docker.cnf

```diff
@@ -1,3 +1,3 @@
 [mysqld]
-skip-host-cache
+SET GLOBAL host_cache_size=0
```

Comment:
It doesn't work with the `SET GLOBAL`

This works for me:

```
[mysqld]
host_cache_size=0
skip-name-resolve

```

Reply:
Yeah, I found it strange that `SET GLOBAL` was here!! Never seen this syntax before, here.

Also, I'm curious if `host_cache_size` is needed, if `skip_name_resolve=1` ?

Reply:
@LaurentGoderre thanks for noticing that, will fix ASAP

Reply:
fixed, marking as resolved